moped-editor/.env-cmdrc

```diff
@@ -13,26 +13,36 @@
     "REACT_APP_MAPBOX_TOKEN": "pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNraWV4dHR0ZjAwNnYyd3FwYjFoNHduaDcifQ.--3vRm2KHq1gh5K_L0pqtA",
     "REACT_APP_NEARMAP_TOKEN": "NWE1ZTA1YTYtOTg4Yy00ZGNhLWFjMzYtMmYxOTI2M2UyODRk",
     "REACT_APP_KNACK_DATA_TRACKER_SCENE": 514,
-    "REACT_APP_KNACK_DATA_TRACKER_VIEW": 3047,
-    "REACT_APP_KNACK_DATA_TRACKER_APP_ID": "6167314778435d001ea3e7cb"
+    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_VIEW": 3047,
+    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_VIEW": 1483,
+    "REACT_APP_KNACK_DATA_TRACKER_APP_ID": "6167314778435d001ea3e7cb",
+    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_ID_FIELD": "field_3998",
+    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_NAME_FIELD": "field_3999",
+    "REACT_APP_KNACK_DATA_TRACKER_CURRENT_STATUS_FIELD": "field_4000",
```

Comment:
This PR establishes a subtle (bad?) naming scheme. It's not great, but I'm not sure how to better smash what could be a single object into ENV variables.

ENV variables that fit [the pattern here](https://github.com/cityofaustin/atd-moped/blob/flh-7925-signal-id-knack-payload/moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js#L88) are expected to contain a knack field as the value and are used to build the body when synchronizing to data tracker.

Reply:
On more consideration, I'm less happy with this. My rule of thumb is that I'm doing something wrong if I'm encoding information into the key of a key, value pair. The human-readable name for the column is extracted from the key via regex. 👎 

I'm tremendously open to any suggestions; I'm not sure what is a better route here beyond some hackery to escape a JSON object into one ENV variable which is brittle and feels like a portability concession.

Reply:
I don't think this is a bad pattern. I can't really suggest a more elegant solution. But I've done something like you're describing organizing Knack keys and fields into a nested JSON. Not the same context as this but maybe it helps? 🤷‍♂️  

https://github.com/cityofaustin/atd-mobile-signals-work-orders/blob/master/src/queries/api.js#L23

Reply:
Ah, TIL more about the cloning process and that it will preserve column names behind the scenes. That will cut down on things to keep track of. Very cool.